### PR TITLE
remove some line breaks in SpecName macros

### PR DESCRIPTION
**Pull request by peterbe**
> What was wrong/why is this fix needed? (quick summary only)

Lots and lots of uses of `SpecName` has a linebreak in the first argument. 
See https://github.com/mdn/yari/pull/3356 for more info. 

> MDN URL of main page changed
Lots

> Issue number (if there is an associated issue)

> Anything else that could help us review it

@sideshowbarker These changes were a couple I started doing manually as I improved my little Python script. Expect more to come.

### CODE REVIEW

**Comment by peterbe** [comment]
@sideshowbarker If you can look over this initial set for a sanity check, I'll re-run my Python script to do another batch. 

Note-to-self; https://gist.github.com/peterbe/446d0e9d11a823663a14bfdafa9e2b5a

File: files/en-us/web/api/globaleventhandlers/onanimationcancel/index.html [hunk 1]
```diff
@@ -100,8 +100,7 @@ <h3 id="CSS">CSS</h3>
 </pre>
 </div>
 
-<p>Leaving out some bits of the CSS that don't matter for the discussion here, let's take
-  a look at the styles for the box that we're animating. First is the box itself, with all
+<p>Leaving out some bits of the CSS that don't matter for the discussion here, let's take a look at the styles for the box that we're animating. First is the box itself, with all
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationcancel/index.html:103]: Looks like this change crept in for some reason — but it doesn’t break anything, anyway

File: files/en-us/web/api/globaleventhandlers/onanimationcancel/index.html [hunk 2]
```diff
@@ -143,8 +142,7 @@ <h3 id="CSS">CSS</h3>
 <h3 id="JavaScript">JavaScript</h3>
 
 <p>Before we get to the animation code, we define a function which logs information to a
-  box on the user's screen. We'll use this to show information about the events we
-  receive. Note the use of {{domxref("AnimationEvent.animationName")}} and
+  box on the user's screen. We'll use this to show information about the events we receive. Note the use of {{domxref("AnimationEvent.animationName")}} and
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationcancel/index.html:145]: Another one that crept in

File: files/en-us/web/api/globaleventhandlers/onanimationiteration/index.html [hunk 3]
```diff
@@ -104,11 +104,9 @@ <h3 id="CSS">CSS</h3>
 </pre>
 </div>
 
-<p>Leaving out some bits of the CSS that don't matter for the discussion here, let's take
-  a look at the styles for the box that we're animating. First is the box itself. We set
+<p>Leaving out some bits of the CSS that don't matter for the discussion here, let's take a look at the styles for the box that we're animating. First is the box itself. We set
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationiteration/index.html:107]: Here too

File: files/en-us/web/api/globaleventhandlers/onanimationiteration/index.html [hunk 4]
```diff
@@ -104,11 +104,9 @@ <h3 id="CSS">CSS</h3>
...
   its size, position, color, and layout. Note that there's nothing there about animation.
-  That's because we don't want the box to start animating right away. We'll add the
-  {{cssxref("animation")}} style later to start animating the box.</p>
+  That's because we don't want the box to start animating right away. We'll add the {{cssxref("animation")}} style later to start animating the box.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationiteration/index.html:109]: And here

File: files/en-us/web/api/globaleventhandlers/onanimationiteration/index.html [hunk 5]
```diff
@@ -159,8 +157,7 @@ <h3 id="JavaScript">JavaScript</h3>
   which is initially zero, which indicates how many iterations of the animation have
   occurred.</p>
 
-<p>The onanimationiteration event handler is then set up. It sets the box's
-  {{cssxref("animation-play-state")}} to "paused", then updates the text displayed in the
+<p>The onanimationiteration event handler is then set up. It sets the box's {{cssxref("animation-play-state")}} to "paused", then updates the text displayed in the
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationiteration/index.html:160]: And here

File: files/en-us/web/api/globaleventhandlers/onanimationstart/index.html [hunk 6]
```diff
@@ -94,11 +94,9 @@ <h3 id="CSS_content">CSS content</h3>
  </pre>
 </div>
 
-<p>Leaving out some bits of the CSS that don't matter for the discussion here, let's take
-  a look at the styles for the box that we're animating. First is the box itself. We set
+<p>Leaving out some bits of the CSS that don't matter for the discussion here, let's take a look at the styles for the box that we're animating. First is the box itself. We set
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationstart/index.html:97]: another

File: files/en-us/web/api/globaleventhandlers/onanimationstart/index.html [hunk 7]
```diff
@@ -94,11 +94,9 @@ <h3 id="CSS_content">CSS content</h3>
...
   its size, position, color, and layout. Note that there's nothing there about animation.
-  That's because we don't want the box to start animating right away. We'll add the
-  {{cssxref("animation")}} style later to start animating the box.</p>
+  That's because we don't want the box to start animating right away. We'll add the {{cssxref("animation")}} style later to start animating the box.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationstart/index.html:99]: And another

File: files/en-us/web/api/globaleventhandlers/onanimationstart/index.html [hunk 8]
```diff
@@ -137,14 +135,12 @@ <h3 id="CSS_content">CSS content</h3>
 }
 </pre>
 
-<p>Since the CSS describes the animation but doesn't connect it to the box, we'll need
-  some JavaScript code to do that.  We'll get to that shortly.</p>
+<p>Since the CSS describes the animation but doesn't connect it to the box, we'll need some JavaScript code to do that.  We'll get to that shortly.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationstart/index.html:138]: Here also

File: files/en-us/web/api/globaleventhandlers/onanimationstart/index.html [hunk 9]
```diff
@@ -137,14 +135,12 @@ <h3 id="CSS_content">CSS content</h3>
...
 
 <h3 id="JavaScript_content">JavaScript content</h3>
 
 <p>Before we get to the animation code, we define a function which logs information to a
-  box on the user's screen. We'll use this to show information about the events we
-  receive. Note the use of {{domxref("AnimationEvent.animationName")}} and
+  box on the user's screen. We'll use this to show information about the events we receive. Note the use of {{domxref("AnimationEvent.animationName")}} and
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/onanimationstart/index.html:143]: And here

File: files/en-us/web/api/globaleventhandlers/ontransitioncancel/index.html [hunk 10]
```diff
@@ -43,9 +43,7 @@ <h3 id="Value">Value</h3>
   <code><var>target</var></code>, where the target object is an HTML element
   ({{domxref("HTMLElement")}}), document ({{domxref("Document")}}), or window
   ({{domxref("Window")}}). The function receives as input a single parameter: a
-  {{domxref("TransitionEvent")}} object describing the event which occurred; the event's
-  {{domxref("TransitionEvent.elapsedTime")}} property's value should be the same as the
-  value of {{cssxref("transition-duration")}}.</p>
+  {{domxref("TransitionEvent")}} object describing the event which occurred; the event's {{domxref("TransitionEvent.elapsedTime")}} property's value should be the same as the value of {{cssxref("transition-duration")}}.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitioncancel/index.html:46]: Another

File: files/en-us/web/api/globaleventhandlers/ontransitioncancel/index.html [hunk 11]
```diff
@@ -67,16 +65,14 @@ <h2 id="Example">Example</h2>
 
 <h3 id="HTML">HTML</h3>
 
-<p>This creates a {{HTMLElement("div")}} which we'll style with CSS below to make into a
-  box that resizes and changes color and such.</p>
+<p>This creates a {{HTMLElement("div")}} which we'll style with CSS below to make into a box that resizes and changes color and such.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitioncancel/index.html:68]: And another

File: files/en-us/web/api/globaleventhandlers/ontransitioncancel/index.html [hunk 12]
```diff
@@ -67,16 +65,14 @@ <h2 id="Example">Example</h2>
...
 
 <pre class="brush: html;">&lt;div class="box"&gt;&lt;/div&gt;
 </pre>
 
 <h3 id="CSS">CSS</h3>
 
-<p>The CSS below styles the box and applies a transition effect which makes the box's
-  color and size change, and causes the box to rotate, while the mouse cursor hovers over
+<p>The CSS below styles the box and applies a transition effect which makes the box's color and size change, and causes the box to rotate, while the mouse cursor hovers over
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitioncancel/index.html:75]: And one more

File: files/en-us/web/api/globaleventhandlers/ontransitionend/index.html [hunk 13]
```diff
@@ -45,13 +45,11 @@ <h3 id="Value">Value</h3>
   where the target object is an HTML element ({{domxref("HTMLElement")}}), document
   ({{domxref("Document")}}), or window ({{domxref("Window")}}). The function receives as
   input a single parameter: a {{domxref("TransitionEvent")}} object describing the event
-  which occurred; the event's {{domxref("TransitionEvent.elapsedTime")}} property's value
-  should be the same as the value of {{cssxref("transition-duration")}}.</p>
+  which occurred; the event's {{domxref("TransitionEvent.elapsedTime")}} property's value should be the same as the value of {{cssxref("transition-duration")}}.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitionend/index.html:48]: Here too

File: files/en-us/web/api/globaleventhandlers/ontransitionend/index.html [hunk 14]
```diff
@@ -45,13 +45,11 @@ <h3 id="Value">Value</h3>
...
 
 <div class="note">
   <p><code>elapsedTime</code> does not include time prior to the transition effect
-    beginning; that means that the value of {{cssxref("transition-delay")}} doesn't affect
-    the value of <code>elapsedTime</code>, which is zero until the delay period ends and
+    beginning; that means that the value of {{cssxref("transition-delay")}} doesn't affect the value of <code>elapsedTime</code>, which is zero until the delay period ends and
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitionend/index.html:52]: And here

File: files/en-us/web/api/globaleventhandlers/ontransitionend/index.html [hunk 15]
```diff
@@ -64,16 +62,14 @@ <h2 id="Example">Example</h2>
 
 <h3 id="HTML">HTML</h3>
 
-<p>This creates a {{HTMLElement("div")}} which we'll style with CSS below to make into a
-  box that resizes and changes color and such.</p>
+<p>This creates a {{HTMLElement("div")}} which we'll style with CSS below to make into a box that resizes and changes color and such.</p>
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitionend/index.html:65]: And here

File: files/en-us/web/api/globaleventhandlers/ontransitionend/index.html [hunk 16]
```diff
@@ -64,16 +62,14 @@ <h2 id="Example">Example</h2>
...
 
 <pre class="brush: html;">&lt;div class="box"&gt;&lt;/div&gt;
 </pre>
 
 <h3 id="CSS">CSS</h3>
 
-<p>The CSS below styles the box and applies a transition effect which makes the box's
-  color and size change, and causes the box to rotate, while the mouse cursor hovers over
+<p>The CSS below styles the box and applies a transition effect which makes the box's color and size change, and causes the box to rotate, while the mouse cursor hovers over
```
Comment (sideshowbarker) [inline files/en-us/web/api/globaleventhandlers/ontransitionend/index.html:72]: And here

**Review by sideshowbarker — APPROVED** [review]
All the changes to the actual SpecName macro instances LGTM. The other changes I flagged all look fine too — just not sure how they crept in

**Comment by peterbe** [comment]
Clearly, my Python script needs work. And THANK YOU for checking.

**Pull request closed without merging**